modules/standard/Tuner.chpl

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2004-2015 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module Tuner {
+
+  // Chapel runtime low-level tuner interface.
+  pragma "insert line file info"
+  extern proc chpl_tuner_session_init(): opaque;
+
+  pragma "insert line file info"
+  extern proc chpl_tuner_session_fini(id: opaque);
+
+  extern proc chpl_tuner_session_loop_span(id:   opaque,
+                                           span: uint(32));
+
+  pragma "insert line file info"
+  extern proc chpl_tuner_get(id:   opaque,
+                             min:  real,
+                             max:  real,
+                             step: real,
+                             init: real): real;
+
+  // Class to encapsulate
+  class Session {
+    var id: opaque;
+
+    proc Session() {
+      id = chpl_tuner_session_init();
+    }
+
+    proc ~Session() {
+      chpl_tuner_session_fini(id);
+    }
+
+    proc loop_span(span) {
+      chpl_tuner_session_loop_span(id, span: uint(32));
+    }
+  }
```

Comment:
I'll add documentation for this function.  The loop_span() function allows a user to specify the number of iterations a loop should execute before sending performance data to the tuner.  Tight loops may need to execute for several iterations before timestamps become a reliable source for performance data.

I'll put together a commit that changes the name of this function.  I'll also make it a sibling of tune().  I'm not sure I can get away with using only one word, though.